src/chapters/03/main.tex

```diff
@@ -193,13 +207,13 @@ \subsection{Process Based Simulation}
 
 \section{Solving with Python}\label{sec:solving-with-python}
 
-In this book we will use the Ciw library in order to conduct discrete event
-simulation in Python.
-Ciw uses the event scheduling approach, which means we must define the system's
-facilities, and then let customers loose to interact with them.
+In this book the Ciw library will be used in order to conduct Discrete Event
+Simulation in Python.
+Ciw uses the event scheduling approach, which means define the system's
```

Comment:
why have you capitalised Discrete Event Simulation?

Reply:
I have no idea.